apps/credential-showcase-api-server/src/types/schema/index.ts

```diff
@@ -14,6 +15,11 @@ import {
 export type Asset = typeof assets.$inferSelect;
 export type NewAsset = typeof assets.$inferInsert & { fileName?: string | null, description?: string | null };
 
+
+// $inferSelect does not respect nullability of fields and the type has every field as required
```

Comment:
we should have this comments already at the top, i suggest that is enough for now. no need to put them more than once